5calls/app/src/main/java/org/a5calls/android/a5calls/AppSingleton.java

```diff
@@ -35,8 +45,19 @@ public DatabaseHelper getDatabaseHelper() {
 
     public FiveCallsApi getJsonController() {
         if (mFiveCallsApi == null) {
-            mFiveCallsApi = new FiveCallsApi(mContext);
+            mFiveCallsApi = new FiveCallsApi(getRequestQueue(mContext), getGson());
```

Comment:
Because nothing else in the app uses the request queue or Outcome GSON I don't think these functions should be in AppSingleton. Instead, let the FiveCallsApi constructor make the RequestQueue and GSON, abstracted away from the rest of the app.

Reply:
We would benefit from using the dependency injection pattern, but since we don't really do any unit testing yet, I'll change it back.